.eslintrc.js

```diff
@@ -27,6 +34,7 @@ module.exports = {
     '@typescript-eslint/explicit-function-return-type': 0,
     'deprecation/deprecation': 1,
     '@typescript-eslint/no-explicit-any': 0, // Ideally we want this on, but we have some types that I'm unfamiliar with typed as `any`.
+    '@typescript-eslint/no-unused-vars': 0,
```

Comment:
Any reason we are turning off the Typescript check in favor of the JS check?

Reply:
I just wasn't sure if I could configure it the same way as the default one. I can change it if you want